marklogic-data-hub-central/ui/src/components/modeling/concept-class-modal/concept-class-modal.tsx

```diff
@@ -22,6 +22,7 @@ type Props = {
   updateConceptClassAndHideModal: (conceptClass: string, description: string) => void;
   updateHubCentralConfig: (hubCentralConfig: any) => void;
   hubCentralConfig: hubCentralConfig;
+  dataModel: Array<any>;
```

Comment:
Any reason we cannot add this to the const of props below?

Reply:
no reason just used it like that but I already changed it